src/tree/SubscriptionTreeItem.ts

```diff
@@ -60,34 +68,80 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
     }
 
     public async createChildImpl(context: ICreateChildImplContext): Promise<AzureTreeItem> {
-        const client: CosmosDBManagementClient = createAzureClient(this.root, CosmosDBManagementClient);
-        const wizardContext: ICosmosDBWizardContext = Object.assign(context, this.root);
+        const wizardContextInitial: IDBAWizardContext = Object.assign(context, this.root);
 
         const promptSteps: AzureWizardPromptStep<ILocationWizardContext>[] = [
-            new CosmosDBAccountNameStep(),
-            new CosmosDBAccountApiStep(),
-            new ResourceGroupListStep()
+            new DatabaseAccountOptionsStep()
         ];
-        LocationListStep.addStep(wizardContext, promptSteps);
-
-        const wizard = new AzureWizard(wizardContext, {
-            promptSteps,
-            executeSteps: [
-                new CosmosDBAccountCreateStep()
-            ],
-            title: 'Create new Cosmos DB account'
+        const wizardInitial = new AzureWizard(wizardContextInitial, {
```

Comment:
It may be overkill to use a separate wizard here since it's only one prompt. 

Is it possible to combine `DatabaseAccountOptionsStep` with the rest of the wizard? That way we would have a back button

Reply:
Yeah we need to have a more in-depth discussion about the entry points for creating a Postgres server vs cosmos account. For example -
- Command palette: If I execute a Cosmos DB command from the command palette, it doesn't make sense for you to give me the option to create a postgres server.
- Plus button in title bar: Perhaps "PostgreSQL" could be added to the prompt where users currently pick between Mongo/SQL/Graph

I propose we keep it simple in this PR (there's plenty of other stuff to review) and ignore those two entry points for now. Instead, how about you just add a "Create PostgreSQL server..." command to the right click menu on subscriptions. I believe that would be the simplest option and should be easy to refactor in the future. That means you would not change `createChildImpl` here and would just add a `createPostgresServer.ts` file to the `commands` folder for now

Reply:
This comment doesn't look resolved yet. It's the most complicated comment IMO, so happy to discuss over a Teams call if you want

Reply:
It's already fixed, I just forgot to click resolve here.

Reply:
It is in the recent commit, I forgot to resolve it here.